### ISSUE TITLE
Log file request

### ISSUE THREAD

**Issue by JohnnyIlRosso** [finndev/PokeBuddy#259]
Great job with your project, it's awesome!!

I would like to save the log of the current game session in a text file. 

Is it possible to add an option to spool directly all the game log in a text file run-time or just save it automatically when you press the stop button?

Have a nice day!

**Comment by cgerdon** [comment]
No, but i usulally Ctrl + c & v the log to excel and quickly scan over it with filters to see whap happend

**Comment by wlwong28** [comment]
Click the dialogue, "Ctrl + A" > "Ctrl + C", then paste in a place you want :)
